src/System.Management.Automation/FormatAndOutput/common/ComplexWriter.cs

```diff
@@ -49,14 +49,24 @@ internal void WriteObject(List<FormatValue> formatValueList)
             // we always start with no indentation
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Custom formatting returning nothing.Newlines are not written if nothing is returned by custom format
```powershell
Expected strings to be the same, but they were different.
Expected length: 2
Actual length:   3
Strings differ at index 2.
Expected: '^^'
But was:  '^^^'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Format-Custom.Tests.ps1: line 504
504:       $ps.Invoke() -replace '\r?\n', "^" | Should -BeExactly $expectedOutput
```